src/main/java/org/broadinstitute/hellbender/utils/genotyper/ReadLikelihoods.java

```diff
@@ -1078,6 +1078,39 @@ private void removeSampleReads(final int sampleIndex, final IntArrayList indexTo
     }
 
     /**
+     * Transform into a multi-sample HashMap backed {@link PerReadAlleleLikelihoodMap} type.
+     * @return never {@code null}.
+     *
+     */
+    public Map<String, PerReadAlleleLikelihoodMap> toPerReadAlleleLikelihoodMap() {
```

Comment:
There's an open issue in gsa-unstable (number 831 there) to remove the PerReadAlleleLikelihoodMap entirely, so I feel weird carrying that over into GATK4. I think the alternative is a Map of ReadLikelihoods, but I'm not sure where the issue originated.  Maybe @vruano is a good person to ask.